.github/workflows/examples-esp32.yaml

```diff
@@ -96,22 +96,65 @@ jobs:
             - name: Build example Bridge App
               timeout-minutes: 10
               run: scripts/examples/esp_example.sh bridge-app
+
+            - name: Uploading Size Reports
+              uses: actions/upload-artifact@v2
+              if: ${{ !env.ACT }}
+              with:
+                  name: Size,ESP32-Examples,${{ env.GH_EVENT_PR }},${{ env.GH_EVENT_HASH }},${{ env.GH_EVENT_PARENT }},${{ github.event_name }}
+                  path: /tmp/bloat_reports/
+
+    esp32_1:
```

Comment:
some comment on why this is happening would be nice - it seems to want to run using 2x runners because ESP32 takes a long time.

This will pay the cost of extra bootstrapping. Is there a way  to cache the bootstrap/checkout instead during the run?

Reply:
ESP32 workflow builds 10 examples, after adding OTA requestor and provider there will be 12 so, a single job would require more time. I split the workflow into two job as per @bzbarsky-apple's [comment](https://github.com/project-chip/connectedhomeip/pull/13212#issuecomment-999577794).

We can cache the files generated by bootstrap, but size of `.environment` is around 3.4GB, so I am not sure whether should we do that or not. Also, I think we'll loose the ability to run the job in parallel.

@andy31415 since this adds extra cost what would you suggest?